terraform-dev/cluster.tf

```diff
@@ -3,7 +3,12 @@ variable "image_name" {
   default = "ubuntu-18.04-server-cloudimg-amd64.img"
 }
 
-variable "disk_size" {
+variable "root_disk_size" {
+  type = string
```

Comment:
Can you add descriptions what the disks will be used for?

Reply:
Done.